### PR TITLE
Improve flow coverage

### PR DESCRIPTION
**Pull request by AlexMSmithCA**
```
#### Problem/Rationale

Fusion.js packages, with the exception of `fusion-core` do not export [libdef](https://flow.org/en/docs/libdefs/) and consumers rely on the Flow type definitions embedded in the source code.  Given this, it is a pain point for consumers when type definitions are missing or incomplete.

We should also strive for full Flow coverage to minimize issues within our own packages.

#### Solution/Change/Deliverable

Two-fold:
* Reach 100% Flow coverage on exported type definitions.
* Strive for 100% Flow coverage internally for each package.
```

screen shot 2018-04-23 at 6 01 02 pm

(compare to [before](https://user-images.githubusercontent.com/3497835/39161635-864f4c9a-4726-11e8-9880-c364b2477162.png))

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -49,9 +52,8 @@ function loadOrGenerateSecret(session) {
   return secret;
 }
 
-const CsrfPlugin =
-  // $FlowFixMe
-  __NODE__ &&
+const plugin: FusionPlugin<CsrfDepsType, CsrfServiceType> =
+  // __NODE__ &&
```
Comment (KevinGrandon) [inline src/server.js]: Don't we still need to keep this?

File: package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
     "lint": "eslint . --ignore-path .gitignore",
     "transpile": "npm run clean && cup build",
     "build-test": "rm -rf dist-tests && cup build-tests",
-    "just-test": "node_modules/.bin/unitest --browser=dist-tests/browser.js --node=dist-tests/node.js",
+    "just-test": "node_modules/.bin/unitest --debug --browser=dist-tests/browser.js --node=dist-tests/node.js",
```
Comment (KevinGrandon) [inline package.json]: What does the debug flag do, should we remove it?

File: src/server.js (hunk 1 above)
Comment (AlexMSmithCA) [reply src/server.js]: Whoops, yup!  We'll want this back.

File: package.json (hunk 2 above)
Comment (AlexMSmithCA) [reply package.json]: Unfortunately, [it does nothing](https://github.com/rtsao/unitest#debug-node-tests) :(.  I'll remove it.

**Pull request merged**